src/internal_events/aws_s3.rs

```diff
@@ -12,29 +10,54 @@ pub mod source {
     use crate::sources::aws_s3::sqs::ProcessingError;
 
     #[derive(Debug)]
-    pub struct SqsS3EventReceived {
+    pub struct SqsS3EventsReceived {
         pub byte_size: usize,
     }
 
-    impl InternalEvent for SqsS3EventReceived {
+    impl InternalEvent for SqsS3EventsReceived {
+        fn emit_logs(&self) {
+            trace!(
+                message = "Events received",
```

Comment:
```suggestion
                message = "Events received.",
```
(for consistency... as an aside, I wonder if these messages and names should've just been defined as constants somewhere)

Reply:
Yeah, agreed, I think constants would help here (and enums for the different error types and stages). That can be a second pass though.